### PR TITLE
Refactor votes to include 2 scores

### PR DESCRIPTION
**Pull request by casassg**
After internal talks regarding what should we take into account when reviewing an application, we decided to do the following. Add 2 scores: Technical and Personal. 

- Standardization of scores for each reviewer.
- Order in admin view for votes
- Filter by university and/or country
- Weighted vote: 0.8 personal + 0.2 tech

 See this for a proper explanation on standardization: http://www.dataminingblog.com/standardization-vs-normalization/

### CODE REVIEW

**Comment by casassg** [comment]
closes #35

**Comment by casassg** [comment]
Not particularly in love with the visualization of the 2 score, they are not centered like the rest. Will probably try a few more options

**Comment by casassg** [comment]
Fixed some problems regarding admin.

hen a queryset was annotated it could update it, so basically it was giving us a 500 beautiful error

To see what's this problem please see this: http://stackoverflow.com/questions/13559944/how-to-update-a-queryset-that-has-been-annotated

File: register/templates/vote.html [hunk 1]
```diff
@@ -63,17 +64,36 @@ <h4 class="text-center">Social</h4>
...
+                                application
+                            </button>
                         </form>
 
 
                     </div>
                 </div>
             {% else %}
-                <h4 class="text-center" style="color: white;">No applications left to be voted.<br> Good job! Enjoy the sun outside!</h4>
+                <h4 class="text-center" style="color: white;">No applications left to be voted.<br> Good job! Enjoy the
```
Comment (ani300) [inline register/templates/vote.html]: Two lines here?

**Review by ani300 — COMMENTED** [review]
Only this, rest of the code LGTM

File: register/templates/vote.html (hunk 1 above)
Comment (casassg) [reply register/templates/vote.html]: true. it was pycharm adding new lines.. weird because it didn't happen before

**Pull request merged**